### PR TITLE
Gate resources behind an email capture

### PR DESCRIPTION
**Pull request by woodhull**
This adds support for gating "resource pages" behind an email subscribe ask. The email opt in is optional. 

 Website copy is placeholder for now. 
 
 Preview:
 https://6036b299a3d0d200074c3db8--cranky-wiles-d63140.netlify.app/resources/getting-started-with-distributed-organizing

### CODE REVIEW

**Comment by woodhull** [comment]
Actually authoring good content is in the next PR / to be done via netlify.

File: _resources/getting-started-with-distributed-organizing.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 ---
 layout: resource
 title: Getting Started with Distributed Organizing
+resource_url: /foo.pdf
```
Comment (anero) [inline _resources/getting-started-with-distributed-organizing.md:4]: This URL is the placeholder you mentioned on the PR details?

File: _layouts/resource.html [hunk 2]
```diff
@@ -1,12 +1,81 @@
...
 layout: default
 ---
+
 <div class="container-fluid bg-white text-black blog">
-    <div class="container">
-        <div class="row justify-content-center py-4">
-            <div class="col-md-8">
-                {{content}}
+  <div class="container">
+    <div class="resource-content d-none" id="resource-content">
```
Comment (anero) [inline _layouts/resource.html:7]: Is this used at all? I don't see `d-none` being override for this section anywhere

File: _src/css/_typography.scss [hunk 3]
```diff
@@ -24,6 +24,10 @@
   font-family: Grafik-Medium, sans-serif;
 }
 
+.regular {
+  font-family: Grafik-Regular, SFProDisplay, Arial, sans-serif, sans-serif;
```
Comment (anero) [inline _src/css/_typography.scss:28]: Duplicated `sans-serif`

**Review by anero — APPROVED** [review]
Looks good, even though email opt-in is optional users still are required to enter an email address, right?

File: _layouts/resource.html (hunk 2 above)
Comment (woodhull) [reply _layouts/resource.html:7]: Yes, it's used by the js by removing the class to show the content.

File: _resources/getting-started-with-distributed-organizing.md (hunk 1 above)
Comment (woodhull) [reply _resources/getting-started-with-distributed-organizing.md:4]: Yes.

**Pull request merged**